app/models/species.rb

```diff
@@ -1,4 +1,4 @@
-class Specie < ApplicationRecord
+class Species < ApplicationRecord
```

Comment:
Yes, Species is for Singular and Plural we got several routes issues with specie and Species, we talked with Kuri about it and decided to let it only with Species